packages/ses/src/intrinsics.js

```diff
@@ -98,25 +98,40 @@ export const makeIntrinsicsCollector = () => {
       if (typeof permit !== 'object') {
         throw TypeError(`Expected permit object at whitelist.${name}`);
       }
-      const namePrototype = permit.prototype;
-      if (!namePrototype) {
+      const permitPrototype = permit.prototype;
+
+      if (
+        typeof intrinsic === 'function' &&
+        intrinsic.prototype !== undefined &&
+        permitPrototype === 'undefined' // permits.js
```

Comment:
it's literally the string `undefined`?

Reply:
Yes. When a permit is a string which is a typeof name, it means "this field must be a value whose typeof is that string. The only value v for which typeof v === 'undefined' is undefined.

Reply:
But, IIUC, given #2624 , none of this may still be necessary, which is why I ask.

Reply:
Or the possibility of just forcing these functions to have no `prototype` property [even in Hermes](https://github.com/endojs/endo/pull/2563#issuecomment-2417858134).

Reply:
after testing https://github.com/endojs/endo/pull/2624 with `git rebase master` on https://github.com/endojs/endo/pull/2334

can confirm https://github.com/endojs/endo/pull/2624 fixes _completePrototypes_ on Hermes and this PR's changes no longer necessary - thank you!

closing this PR with comment of the results